modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -15,88 +14,56 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 .Procedure
 
-. Create a new file for the network attachment definition in any local directory.
-The file must have the following contents, modified to match your
-configuration:
+. Create a network attachment definition manifest.
+The manifest must have the following contents, modified to match your configuration:
 +
 [source,yaml]
 ----
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: a-bridge-network
+  name: <a-bridge-network> <1>
   annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <1>
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/<br0> <2>
```

Comment:
This might only be me not familiar with the rules of documentation, but to my understanding - the triangular parentheses <> are not supposed to include an actual value.`br0` is an actual name of a node bridge interface, especially when you say in bullet #2 (Line 45): "...run on nodes that have the `br0` bridge connected".

Reply:
Yeah, we use angle brackets to show replaceable values. In this case I thought that the bridge 'br0' is not a fixed name and should be denoted as replaceable. 
I've kept the angled brackets in the YAML but updated the callout description:
<2> Optional: Annotation key:value for node selection, where `br0` is the name of a bridge configured on some nodes. If you add this annotation to your network attachment definition, your virtual machine instances will only run on the nodes that have the `br0` bridge connected.
Does that help?

Reply:
Yep, that nails it perfectly. Thanks Andrew.